tclreadlineSetup.tcl.in

```diff
@@ -289,6 +289,11 @@ namespace eval tclreadline {
                 if {[string length $::tclreadline::LINE]
                         && [history event 0] != $::tclreadline::LINE} {
                     history add $::tclreadline::LINE
+                    if {[info exists ::tclreadline::autosave] && $::tclreadline::autosave == 1 && \
```

Comment:
How about `[string is true -strict $::tclreadline::autosave]` so that boolean values of 't' or 'f' can also be understood?